libs/wingsdk/src/cloud/bucket.ts

```diff
@@ -35,6 +35,11 @@ export abstract class BucketBase extends Resource {
 
     props;
   }
+
+  /**
+   * Add a file to the bucket that is uploaded when the app is deployed.
+   */
+  public abstract addObject(key: string, body: string): void;
```

Comment:
I did not see one but I wonder if we have any issue that references future support of blobs. Maybe a TODO: with the issue link might mitigate questions from those who wonder why we are using just string for body. 🤷

Reply:
I had the same question, but I thought that since the language didn't have blob support yet that's why str was used

Reply:
I've added a comment to the description 👍